src/irr/asset/COBJMeshFileLoader.cpp

```diff
@@ -466,6 +407,19 @@ asset::SAssetBundle COBJMeshFileLoader::loadAsset(io::IReadFile* _file, const as
             vtxBufBnd.offset = 0ull;
             vtxBufBnd.buffer = vtxBuf;
             submeshes[i]->setVertexBufferBinding(std::move(vtxBufBnd), BND_NUM);
+
+			if (recalcNormals[i])
+			{
+				auto vtxcmp = [&vtxSmoothGrp](const IMeshManipulator::SSNGVertexData& v0, const IMeshManipulator::SSNGVertexData& v1, ICPUMeshBuffer* buffer)
+				{
+					static constexpr float cosOf45Deg = 0.70710678118f;
+					return vtxSmoothGrp[v0.indexOffset]==vtxSmoothGrp[v1.indexOffset] && 
+						core::dot(v0.parentTriangleFaceNormal, v1.parentTriangleFaceNormal)[0] > cosOf45Deg;
```

Comment:
when OBJ has smooth groups, are you even supposed to check for crease angle?

Reply:
hm good point, probably not